### DISCUSSION TITLE
Using a mojolicious plugin outside of an app (TagHelpers)

### DISCUSSION THREAD

**Discussion by daleif** [mojolicious/mojo#1853] [Q&A] [answered]
Perhaps strange use case...

I had to generate some HTML code to paste into a CMS (it was just easier that way), so wrote a script using Mojo::Template

I happened to want to use some of the helpers from the TagHelpers plugin. But how can we use this with the template engine when we are not even writing an app?

Is there something easy I'm missing here? (note my call to the template is just `$mt->vars(1)->render($template, $data)` )

**Comment by Grinnz** [comment, marked as answer]
You could load a dummy app and use its renderer (untested):

```
my $app = Mojo::HelloWorld->new;
$app->renderer->paths([$template_dir]);
my $output = $app->build_controller->render_to_string('template_name');
```
